src/routes/room.route.js

```diff
@@ -2,5 +2,9 @@ const router = require('express').Router()
 const roomController = require('../controllers/room.controller')
 
 router.post('/create', roomController.create);
+router.get('/all', roomController.getAll);
+router.get('/:guid', roomController.getById);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Backend/pullRequest?prid=7180713)
```suggestion
router.get("/:guid", roomController.getById);
```